printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md

```diff
@@ -0,0 +1,14 @@
+# BTT octopus vertical mount
+
+## Overview
+This is a mount for new BTT octopus mount for rail. it mounts the board vertically to take full advantage of the fan to cool those stepper motor
+
+remember to use the spacer.
```

Comment:
Capitalize `Remember` at the start of the sentence.

`spacer` should probably be plural `spacers` if you really need 4 of them.

You should mention here where the spacer should be used.

Reply:
Please use plural `spacers`, since there are 4 of them.